README.rst

```diff
@@ -100,14 +105,43 @@ To enable this plugin:
     Scrapy setting to
     ``"twisted.internet.asyncioreactor.AsyncioSelectorReactor"``.
 
+In both cases:
+
 -   Set `your Zyte API key
     <https://docs.zyte.com/zyte-api/usage/general.html#authorization>`_ as
     either the ``ZYTE_API_KEY`` Scrapy setting or as an environment variable of
     the same name.
 
+For example, in the ``settings.py`` file of your Scrapy project:
+
+.. code-block:: python
+
+    ADDONS = {
+        "scrapy_zyte_api.Addon": 1,
+    }
+    ZYTE_API_KEY = "YOUR_API_KEY"
+
+or:
+
+.. code-block:: python
+
+    DOWNLOAD_HANDLERS = {
+        "http": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler",
+        "https": "scrapy_zyte_api.ScrapyZyteAPIDownloadHandler",
+    }
+    DOWNLOADER_MIDDLEWARES = {
+        "scrapy_zyte_api.ScrapyZyteAPIDownloaderMiddleware": 1000,
+    }
+    REQUEST_FINGERPRINTER_CLASS = "scrapy_zyte_api.ScrapyZyteAPIRequestFingerprinter"
+    TWISTED_REACTOR = "twisted.internet.asyncioreactor.AsyncioSelectorReactor"
+    ZYTE_API_KEY = "YOUR_API_KEY"
```

Comment:
The readme structure changed a bit, so now this example is a duplicate of the quickstart section. What do you think about moving ADDONS... example to the quickstart as well?